libraries/customLintLibrarySpiAuto/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: "java-library"
+apply plugin: "application"
+
+dependencies {
+    annotationProcessor deps.apt.autoService
+    compileOnly deps.apt.autoService
+    compileOnly deps.lint.lintApi
+    compileOnly deps.lint.lintChecks
+
+    testImplementation deps.lint.lint
+    testImplementation deps.lint.lintTests
+}
+
+okbuck {
+    containsLintRegistry = true
```

Comment:
This is configured currently at the top level okbuck extension. Let's keep the same behavior for consistency?

Reply:
See my comment on the PR: How would we mark a specific module as such if it's only defined all at root level? I'm thinking we should just use a gradle property instead